index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Agasha Bornittah Portifolio</title>
+    <link rel="stylesheet" href="assets/fonts/css/all.min.css">
+    <link rel="stylesheet" href="assets/fonts/css/brands.min.css">
+    <link rel="stylesheet" href="assets/fonts/css/fontawesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+   <div class="container">
+       <div class="nav">
+        <h1>Welcome</h1>
+        <div class="nav-menu" >
+            <ul>
+                <li>
+                    <a href="">Portifolio</a>
+                </li>
+                <li>
+                    <a href="">About</a>
+                </li>
+                <li>
+                    <a href="">Contact</a>
+                </li>
+            </ul>
+        </div>
+        <a href="" class="menu-bars">
+            <i class="fas fa-bars"></i>
+        </a>
+       </div>
```

Comment:
- [ ] Kindly change the div tag wrapping the logo and hamburger icon to be a header tag as this is part of the project requirements 👍🏿